charts/ingress-nginx/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: ingress-nginx
 version: 2.6.0
-appVersion: 0.33.0
+appVersion: 0.34.0
```

Comment:
There is no 0.34.0. Keep the original value

Reply:
sorry, version now updated instead of appVersion